tests/config.example.json

```diff
@@ -1,8 +0,0 @@
-{
-  "applicationName": "deno_postgres",
```

Comment:
Sweet - I bet you could ditch the readme under tests and just mention that docker and docker compose are required.